14/umbraco-cms/fundamentals/data/relations.md

```diff
@@ -8,69 +8,42 @@ Umbraco sections are built around the concept of 'trees' and there is an implici
+In some cases there are no direct relationships between two items in a tree, but they are still somehow 'related'. This could be the alternate language translation pages of a content page.
 
-What if there is also a 'relation' between different types of Umbraco entities in your site, eg Content -> Member or Member -> MediaFolder? Perhaps, you'd like to be able to retrieve and display the uploaded images from a specific logged-in Member.
+In other cases there is a 'relation' between different types of entities. This could be a relation between Content and Member, or Member and MediaFolder. You might need to be able to retrieve and display the uploaded images from a specific logged-in Member.
 
 These are the scenarios where the concept of **Umbraco Relations** provides a solution.
 
 ## The Concept of Umbraco Relations
 
-Umbraco Relations allow you to relate almost any object in Umbraco to almost any other Umbraco object - under a defined _Relation Type_.
+Umbraco Relations allow you to relate almost any object in Umbraco to almost any other Umbraco object. This is done by defining a new _Relation Type_.
 
 ### How is this different to pickers?
 
-With a Content/Member/Media picker that you might add to a Content Type - the relationship is ONLY 1-way. Your content item knows it has 'picked' another content item, perhaps to display a shared banner across different pages, but that 'banner' content item doesn't know where it has been picked!
+With a Content, Member or Media picker the relationship only works as a 1-way street. The content item knows it has 'picked' another content item but that other content item does not know where it has been picked.
 
-Umbraco Relations are 2-way, when you create a relation between two different types of entities, you can alway find one entity from the other and vice versa. eg you could list out in the backoffice all the pages that a content banner had been picked on.
+Umbraco Relations works as a 2-way street. When creating a relation between two different types of entities, it will be possible to find one entity from the other and vice versa. As an example this provides the option to list out all the pages that a content banner had been picked on.
 
 ## Relation Types
 
-To create and use Relations in your Umbraco Website, you need to define a 'Relation Type' to specify what the two types of entities will be relatable and to give the Relation Type an alias. So, you can test if two items are related for a particular 'relation type'. Two items might be related under multiple different Relation Types, and you might be only interested in your 'Related Language Page' Relation Type.
+To create and use Relations in your Umbraco Website, you need to define a 'Relation Type'. The Relation Type will specify what two types of entities will be relatable. Two items might be related under multiple different Relation Types, and you might be only interested in your 'Related Language Page' Relation Type.
```

Comment:
Umm.. I don't think we can define a Relation Type anymore from the Backoffice?

Reply:
True 🤔 
I think I'll try to rephrase the entire section.

Reply:
Looks good to me.